packages/modules/pricing/integration-tests/__tests__/services/pricing-module/price-set.spec.ts

```diff
@@ -422,6 +437,40 @@ moduleIntegrationTestRunner({
               ],
             })
           )
+
+          const [priceRules] = await service.listPriceRules({
+            price_set_id: [priceSet.id],
+          })
+
+          expect(eventBusEmitSpy.mock.calls[0][0]).toHaveLength(3)
+          expect(eventBusEmitSpy.mock.calls[0][0][0]).toEqual(
```

Comment:
Those [0][0][0] are so ugly to read 😄 Maybe the mock event bus can expose some helper methods for asserting calls instead?

Reply:
normally we just use `toHaveBeenCalledWith`, which is already the helper to do that

Reply:
The debugging experience with `toHaveBeenCalledWith` is not great. When you give it an array of events and one of the events mis-match, then there is no feedback on which index there is a mis-match.

Whereas with individual assertion calls, you know exactly which assertion failed